packages/social-previews/src/facebook-preview/style.scss

```
@@ -189,3 +189,22 @@
 		filter: invert(8%);
 	}
 }
+
+.facebook-preview__window {
+	margin: 0 0.75rem;
+
+	/* stylelint-disable-next-line scales/radii */
+	border-radius: 16px;
+	border: $facebook-preview-body-border;
+
+	.facebook-preview__image,
+	.facebook-preview__custom-image {
+		border-top-right-radius: inherit;
+		border-top-left-radius: inherit;
+
+		img {
+			border-top-right-radius: inherit;
+			border-top-left-radius: inherit;
+		}
```

RTL aware alternative styling.

```suggestion
		border-start-end-radius: inherit;
		border-start-start-radius: inherit;

		img {
			border-start-end-radius: inherit;
			border-start-start-radius: inherit;
		}
```